doc/getting_started/installation_linux.rst

```diff
@@ -85,6 +85,24 @@ Clear Linux:
    sudo swupd bundle-add c-basic dev-utils dfu-util dtc \
      os-core-dev python-basic python3-basic python3-tcl
 
+The Clear Linux focus is on *native* performance and security and not
+cross-compilation. For that reason it uniquely exports by default to the
+:ref:`environment <env_vars>` of all users a list of compiler and linker
+flags. Zephyr's CMake build system will either warn or fail because of
+these. To clear the C/C++ flags among these and fix the Zephyr build, run
+the following command as root then log out and back in:
+
+.. code-block:: console
+
+   # echo 'unset CFLAGS CXXFLAGS' >> /etc/profile.d/unset_cflags.sh
```

Comment:
Can't we add this to `~/.zephyrrc` file instead?

Reply:
Yes if you find a reliable way to source it (or zephyr-env.sh) that doesn't get overriden. So this doesn't change the problem, it just moves it somewhere else.
I'm already linking to https://docs.zephyrproject.org/latest/guides/env_vars.html which is the best place to discuss all environment questions once for all; however I could insist a bit more on that link, would you like me to?